imp/TokenSet.cs

```diff
@@ -175,6 +175,20 @@ private TokenSet()
             // no operation
         } /* end TokenSet */
 
+        /* ---------------------------------------------------------------------------
+        * private constructor TokenSet (uint[])

```

Comment:
We already have constructor methods `newFromList()` and `newFromUnion()`. As you can see in line 170, I have made the default constructor `TokenSet()` private in order to prevent clients of the class from ever being able to   use any other constructors than `newFromList()` and `newFromUnion()`.

Your comment says "private" but your code declares it public. But even as a private method it makes no sense because the two constructors `newFromList()` and `newFromUnion()` use the private default constructor in lines 201 and 248.